masschallenge/impact-api
# AC-7771 - get (most) tests passing on API app

## [AC-7771](https://masschallenge.atlassian.net/browse/AC-7771)
### Also see: [accelerate PR](https://github.com/masschallenge/accelerate/pull/2627)
### Testing
This branch does not function standalone like legacy impact-api, so `make test` in impact-api will not work, and Travis test runs won't either.

1. Check out the AC-7771 branch on accelerate, **and in the submodule** `mcproject/api`
2. Run `make build` in accelerate (most importantly this installs dependencies that the API needs)
3. In the accelerate root, run `make test tests=api.web.impact.impact.tests`
3a. You can of course narrow that to a specific test suite, e.g. `api.web.impact.impact.tests.test_graphql`

### Remaining test errors/failures
As of this PR (and its accelerate sibling) there are 8 tests throwing errors, and 9 failing. Shankar agreed with my suggestion to put those into a follow-on ticket: [AC-7827](https://masschallenge.atlassian.net/browse/AC-7827).
